.github/workflows/containers/github-action-ci/Dockerfile

```diff
@@ -103,3 +103,14 @@ RUN mkdir actions-runner && \
     tar xzf ./actions-runner-linux-x64-$GITHUB_RUNNER_VERSION.tar.gz && \
     rm ./actions-runner-linux-x64-$GITHUB_RUNNER_VERSION.tar.gz
 
+# Pre-cache llvm-project in these images; this works with the premerge action
+# to speed up runs; cloning LLVM fresh takes >1min otherwise.
+#
+# A local experiment showed an explicit `git gc --aggressive` reduced this
+# layer's size by 20% (~800MB). Disable GC'ing afterward, so we don't waste time
```

Comment:
Is the layer size here 800MB or was the savings 800MB? I'm assuming the former, but might help to make that a bit more clear.

Reply:
Good point. Savings was 800MB; this layer is 3.29GB on my most recent build. Fixed the wording up as requested - PTAL